### ISSUE TITLE
Quest Grammar in Hawk's Vigil

### ISSUE THREAD

**Issue by ahhh-reptar** [slowtorta/turtlewow-bug-tracker#596]
At multiple points in the quest descriptions, the grammar is pretty rough around the edges.  Some of it is just wrong, some of it just feels off.  I missed some of it just because I wasn't writing stuff down all the way until just now.  This area in general also uses a LOT of adverbs which is generally something you want to avoid in creative writing, but it's more forgivable during character speech.

The Mystery Continues-
Wrong: "the mystery still remains at foot" - at foot should be afoot.
Wrong: "his time is much more occupied then mine" - then should be than.

Tinkering Away-
Wrong: "I was renowned engineer" should be "I was a renowned engineer."
Wrong: "gnomish" should be "Gnomish"
Off/Wrong: "I always put my own twists on them for extra sturdiness and to be able to handle a dwarf!"  I think this is meant to say his inventions are fit for a Dwarf, but I'm not sure.

Zabini's Information-
Wrong: "It's presence" should be Its.
Wrong: "Or at the very least has only cropped up at the last few weeks at a level where many like myself, and Marge would have come to notice it." is a sentence fragment.  Belongs attached to previous sentence, and there shouldn't be a comma after myself.  You could just do a dash or nothing to connect it to the last paragraph.
Wrong: "or perhaps something, or someone," or someone should be separated by hyphens or parentheses, though usually it's hyphens.
Wrong: "I wish you luck in finding what is causing the tainting darkness," Slightly odd phrasing, but the important thing here is the following statement is only loosely associated with this one, so it should be separated by either a semicolon or a period.  A semicolon is what your college professor would give you points for here though.
Wrong: "re-live" should be relive.

Zabini's Information (on completion)-
Wrong: "Hmm.." An ellipsis explicitly has 3 periods.  
Off: "So he believes that perhaps this source" is wordy.  I'd take another swing at it.  It takes too long to reach the goal.

A Potential Clue-
Wrong: "So, Zabini may be on to something actually," Use onto in all circumstances unless on is part of the verb.  Here be is the verb, so onto shows his ability to understand.  Also, don't end this phrase with a comma, end it with a period and start the next sentence fresh.
Wrong: "in all my research I have been unable to locate the source of this magic," End this with either a period or a semicolon, same explanation as before.
Wrong: "But," remove the comma.  Also, later in this sentence, he is somewhat unclear.  Consider replacing it with a proper noun, or rearranging the last paragraph to bring Zabini closer to "he"
Wrong: "coming's and going's" Remove the apostrophes.  They are neither short for nor possess anything.
Off: "look to them as a potential source of this darkness, or at the very least, investigate it" Kinda repeating herself here, also remove the comma after least, also rephrase it to them (or something more clear).

Lord Commander Ryke (outside of quest)-
Wrong: "comin."  Either put a g or an apostrophe after comin. Both times.

A Potential Clue (on completion)-
Wrong: "Interesting report," change the comma to a period.
Off: "let me check my records and make sure about the new people within the town." Phrased weird.  Make sure what?  Make sure he knows them?  Make sure he knows where they are?  Could be more clear.

Overlapping Investigations-
Wrong: Probably include a period at the end of the first sentence.  I get it's him emoting but still.
Wrong: "So, it seems" Remove the comma.
Off: "into town, though he hails from the..." The statement following "into town," isn't contradicting the last one.  Understandable but still odd phrasing.  Could be cleaner.
Wrong: "Stormwind, Harlus Ashbuckle" Replace the comma with a hyphen.
Off: "out of my jurisdiction to press" Press charges?  What?  Unsure what he means here.
Wrong: "being that this individual has... Complications."  An ellipsis does not ALWAYS end a sentence.  Complications should not be capitalized here.
Wrong: "by the SI:7, I only" either use a period or a semicolon here, not a comma, but probably a period because of how long the next chunk is.
Off: "Meet up with him" is unclear.  The last him was referring to the guy I'm trying to dig up dirt on, not the SI:7 operative.  Use a proper noun here.  Also end this chunk with a period.
Wrong: "Dursley, he can" put an and here, and you can probably drop the extra he to avoid using it so much in close proximity.
Wrong: (on completion) "understand, playername?" Drop the comma.

Robb's Report-
Off: "If Lord Commander Ryke's suspicions be true," is he a pirate?
Wrong: "would be breaking vow he has sworn" "breaking vow" should be "breaking the vow"
Off: "to nobility within Stormwind" I know what's being said but it could be a lot more clear.  Probbaly something like "to the nobility of Stormwind" or something like that.
WAY off: "Of course, that same nobility having gotten him out of the Stormwind Stockade when the man was wanted for crimes of practicing forbidden magic, and murder both" Not a sentence at all.  I'll try to rephrase it here: "Of course, that's the same nobility that got him out of the Stockades despite his crimes including the practice of forbidden magic and murder."  Something like that.  You can just straight rip my sentence if you want, I don't care.
Off: "As much as I have my dislike for the House of Nobles I do what I am told."  Unclear, buried the action of the sentence, try something like, "Despite my malcontent for the House of Nobles, I do what I am told."
Off: "leader of the SI:7 within Stormwind in Old Town" most people would probably just say something to the effect of "leader of the SI:7 within the Stormwind Old Town" or "leader of the SI:7 within Old Stormwind" or something like that.
Wrong: "Keep them sealed," use a semicolon instead of a comma.
Off: "don't show yourself to him, or myself again." use hyphens to set off "or myself", also be more clear who him is.  Shaw?  Genuinely uncertain but that's my best guess.

At this point I'm being sent off to Stormwind so I'll make a new post for when I get there.  This is already a lot.

**Comment by Kruxis** [comment]
Thanks for your report! 

The issues will be fixed on the next restart.

Safe Travels =)
